lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
If we want to support people generally using CLS inside our promise chains/results we need to support a user supplied namespace.